src/Stryker.Core/Stryker.Core/Options/StrykerOptions.cs

```diff
@@ -8,278 +8,225 @@
+    /// These files are ignored from detecting changes in test projects.
+    /// Context: When using the 'since' feature, all tests are run again if files in the test project change (as these could impact the test results)
+    /// When the file is present in this option the tests should not run again as the file does not impact test results.
+    /// </summary>
+    public IEnumerable<ExclusionPattern> DiffIgnoreChanges { get; init; } = Enumerable.Empty<ExclusionPattern>();
+
+    /// <summary>
+    /// When no previous report can be found for the 'since' feature, this commitish is used to se a baseline.
+    /// </summary>
+    public string FallbackVersion { get; init; }
+
+    /// <summary>
+    /// When publishing to the stryker dashboard, sets the project name on the dashboard.
+    /// </summary>
+    public string ModuleName { get; init; }
+
+    /// <summary>
+    /// When set to a value, reports should be opened by the filesystem after the reports have been generated.
+    /// </summary>
+    public ReportType? ReportTypeToOpen { get; init; }
+
+    /// <summary>
+    /// Files that should be mutated or ignored. Uses GLOB as pattern matching. Also parts of files can be ignored by format {10..21}
+    /// </summary>
+    public IEnumerable<FilePattern> Mutate { get; init; } = [FilePattern.Parse("**/*")];
+
+    /// <summary>
+    /// Method call mutations that should not be tested. The implementation of the method may still be mutated and tested.
+    /// </summary>
+    public IEnumerable<Regex> IgnoredMethods { get; init; } = [];
+
+    /// <summary>
+    /// Mutations that should not be tested.
+    /// </summary>
+    public IEnumerable<Mutator> ExcludedMutations { get; init; } = [];
+
+    /// <summary>
+    /// Linq expressions mutations that should not be tested.
+    /// </summary>
+    public IEnumerable<LinqExpression> ExcludedLinqExpressions { get; init; } = [];
+
+    /// <summary>
+    /// The optimization mode for coverage analysis for the current run.
+    /// </summary>
+    public OptimizationModes OptimizationMode { get; init; }
+
+    /// <summary>
+    /// This name is used in the dashboard report
+    /// Is settable because this version can be detected by using DotNet.ReproducibleBuilds and thus can be overridden by stryker internally
+    /// </summary>
+    public string ProjectName { get; set; }
+
+    /// <summary>
+    /// This projectversion is used in the dashboard report
+    /// Is settable because this version can be detected by using DotNet.ReproducibleBuilds and thus can be overridden by stryker internally
+    /// </summary>
+    public string ProjectVersion { get; set; }
+
+    /// <summary>
+    /// Instruct Stryker to break execution when at least one test failed on initial run.
+    /// </summary>
+    public bool BreakOnInitialTestFailure { get; set; }
+    
+    // Keep a reference on the parent instance in order to flow get/set properties (ProjectName and ProjectVersion) up to the parent
```

Comment:
The comments for the Copy mentions that this is required for the dashboard reporter. You've kept the comment but removed the implementation. Are you certain that this is no longer necessary for the correct working of the dashboard reporter?

Reply:
Build and tests are green, so I assume it is a fair assumption that this is no longer required. I do not think there is some kind of run time discovery for this method, at least I do not see anything that looks like it in the dashboard reporter.
This method was flagged as no longer used, so I removed it on an opportunity basis.

If my memory serves me well, this dates from the initial multiple project/solution mode implementation, when it was handled at a very high level in Stryker, and options were copied and adjusted for each project.
This was an issue for the dashboard reporter (obviously) which could not identify which project was the reference.
This copy logic has been removed for a long time now.